docs/api/migration-guides/_toc.json

```diff
@@ -35,16 +35,28 @@
         "title": "Migrate to Qiskit Runtime",
         "children": [
           {
-            "title": "How to migrate",
+            "title": "Overview",
             "url": "/api/migration-guides/qiskit-runtime"
           },
           {
-            "title": "Examples",
-            "url": "/api/migration-guides/qiskit-runtime-examples"
+            "title": "Migrate from qiskit-ibmq-provider",
+            "url": "/api/migration-guides/qiskit-runtime-from-ibmq-provider"
+          },
+          {
+            "title": "Migrate from qiskit-ibm-provider",
+            "url": "/api/migration-guides/qiskit-runtime-from-ibm-provider"
+          },
+          {
+            "title": "Migrate options",
+            "url": "/api/migration-guides/qiskit-runtime-options"
+          },
+          {
+            "title": "Common use cases",
+            "url": "/api/migration-guides/qiskit-runtime-use-case"
           },
           {
-            "title": "qiskit_ibm_provider to qiskit_ibm_runtime",
-            "url": "/api/migration-guides/qiskit-runtime-from-provider"
```

Comment:
This breaks prior URLs unless we set up a redirect. Is it possible to go back to the old URL?

Reply:
@Eric-Arellano We could; but the URL won't match the new title.  Maybe that's OK though.  @jyu00 ?